src/foam/nanos/dig/DUGRule.js

```diff
@@ -0,0 +1,103 @@
+foam.CLASS({
+  package: 'foam.nanos.dig',
+  name: 'DUGRule',
+  extends: 'foam.nanos.ruler.Rule',
+
+  requires: [
+    'foam.nanos.http.Format'
+  ],
+
+ tableColumns: [
+    'id',
+    'documentation',
+    'enabled',
+    'priority',
+    'daoKey',
+    'operation',
+  ],
+
+  searchColumns: [
+    'id',
+    'url',
+    'enabled',
+    'priority',
+    'daoKey',
+    'operation',
+    'after',
+    'validity'
+  ],
+
+  properties: [
+    {
+      name: 'id',
+      visibility: 'RW'
```

Comment:
We would rather not exposing `id` to users who have access to DUG rule since ones can accidentally or intentionally overwrite existing rules.

Please create a new property called name and remove 'RW' access from `id`.